pkgs/development/python-modules/systembridge/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+}:
+
+buildPythonPackage rec {
+  pname = "systembridge";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "timmo001";
+    repo = "system-bridge-connector-py";
+    rev = "v${version}";
+    sha256 = "1r6b7nh0ni4485l841mh1p6ihqc3iqv0nagqxxiafc3lb5v4y52g";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+  ];
+
+  patches = [
+    # https://github.com/timmo001/system-bridge-connector-py/pull/6
+    (fetchpatch {
+      name = "fix-module-not-found.patch";
+      url = "https://github.com/timmo001/system-bridge-connector-py/commit/342f3033aaf847345eb3b62f00358b53d078c1b4.patch";
+      sha256 = "1jba1dkazm3qnj720cv6xspr3rrpcpvblbvv53n68brkxgg8b1gs";
+    })
+  ];
```

Comment:
If you update to 1.1.1 we can remove the patch.

Reply:
Update to 1.1.1